app/src/main/java/com/infomaniak/mail/ui/main/thread/actions/ThreadActionsBottomSheetDialog.kt

```diff
@@ -96,6 +98,10 @@ class ThreadActionsBottomSheetDialog : ActionsBottomSheetDialog() {
                     mainViewModel.toggleFavoriteStatus(threadUid)
                 }
 
+                override fun onSpam() {
```

Comment:
Extract the possibility to Spam a Thread in another PR, this one is about Phishing.